torch_xla/csrc/debug_util.cpp

```diff
@@ -209,4 +212,43 @@ bool DebugUtil::ExperimentEnabled(const std::string& name) {
   return xset->find(name) != xset->end();
 }
 
+void DebugUtil::analyze_graph_execution_python_frame() {
+  // TODO: maybe only print this for the master process
+  static std::string debug_output_prefix = "Execution Analysis: ";
+  std::vector<torch::lazy::SourceLocation> frames =
+      torch::lazy::GetPythonFrames();
+  // python frame must be > 1
+  XLA_CHECK_GE(frames.size(), 1);
+  std::stringstream ss;
+  ss << debug_output_prefix
+     << "======================================================================"
+        "=========="
+     << "\n";
+  if (frames[0].function == "mark_step") {
```

Comment:
Wondering if it worths logging `frames[0].function;` for the cases other than `mark_step`.

Reply:
yea I want to expand this a bit later to cover most of the common cases of the execution.